.github/workflows/mirror.yml

```
@@ -0,0 +1,34 @@
+name: Mirror code on D.org
+on:
+  push:
+
+concurrency:
+  group: 'ci-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+
+  mirror:
+    runs-on: ubuntu-latest
+    name: Checkout & push to d.org mirror
+    if: ${{ github.repository == 'pantheon-systems/pantheon-content-publisher-drupal' }}
+    env:
+      BRANCH: ${{ github.ref_name }}
+      WORKSPACE: ${{ github.workspace }}
+      DRUPAL_ORG_REMOTE: ${{ secrets.DRUPAL_ORG_REMOTE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

```suggestion
      - name: Checkout
        uses: actions/checkout@v4
```


Let's add a dependabot config for actions w/ this as we get started.

c/f
https://github.com/pantheon-systems/plugin-pipeline-example/blob/main/.github/dependabot.yml#L11-L14

---

@pwtyler I added the dependabot.yml file. Could you review my changes again?
Thank you.